### PR TITLE
xserver: dwm as a window manager

### PR DESCRIPTION
**Pull request by zenhack**

### CODE REVIEW

File: nixos/modules/services/x11/window-managers/dwm.nix [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+in
+
+{
+
+  ###### interface
+
+  options = {
+
+    services.xserver.windowManager.dwm.enable = mkOption {
```
Comment (jagajaga) [inline nixos/modules/services/x11/window-managers/dwm.nix]: Please use `mkEnableOption`.

Comment (zenhack) [inline nixos/modules/services/x11/window-managers/dwm.nix]: Quoting Arseniy Seroka (2015-12-03 08:27:59)

> > +    services.xserver.windowManager.dwm.enable = mkOption {
>
>    Please use mkEnableOption.

Can do. I based this off of another one of the files in that directory,
which still uses mkOption. I gather this is a deprecated way of doing
things that hasn't be cleaned up everywhere? Would a (separate) patch
that updates the rest be welcome?

Comment (jagajaga) [inline nixos/modules/services/x11/window-managers/dwm.nix]: Yes! That's going to be great work.

Comment (zenhack) [inline nixos/modules/services/x11/window-managers/dwm.nix]: I've made the change you requested.

**Comment by AndersonTorres** [comment]
Hum, so far so good. What about a fat commit changing all relevant files? I will do it now.

**Comment by zenhack** [comment]
The travis failure seems to be unrelated to my change; error messages are talking about virt-manager and glance.

the mkEnableOption looks to be all over the source tree, not just in window-managers.

**Pull request merged**